bitcoin/src/pow.rs

```diff
@@ -1908,18 +1907,23 @@ mod tests {
 
     #[test]
     fn target_difficulty_float() {
-        assert_eq!(Target::MAX.difficulty_float(), 1.0_f64);
+        let params = Params::new(crate::Network::Signet);
```

Comment:
Please use Mainnet (mentioned in previous comment) since the literal values for the test are easily human verified by just reading them.

You can copy paste these tests to add Signet in addition to Mainnet though.

Reply:
Ohh, By mistake I used `Signet` instead of `Mainnet` -> thanks for pointing it.

> You can copy paste these tests to add Signet in addition to Mainnet though.

I feel -> doing test only for  Mainnet meets the requirements -> no need to add tests for Signet also.